securitysystem/thingspeakreader.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+"""
+Reading data collected from ThingSpeak
+"""
+import requests
+import json
+import logging
+import argparse
+import constants as c
+
+class ThingSpeakReader():
```

Comment:
I wonder how different our ThingSpeak classes are... If we have time to compare we might be able to combine everything into a common class

Reply:
I don't think we have much time left so might as well leave them separate, or else if we start combining them and there are bugs we will run into more problems

Reply:
I agree, it can be something we talk about in our report instead maybe